src/main/java/me/totalfreedom/totalfreedommod/command/Command_myadmin.java

```diff
@@ -17,7 +17,7 @@
 import org.bukkit.entity.Player;
 
 @CommandPermissions(level = Rank.ADMIN, source = SourceType.ONLY_IN_GAME)
-@CommandParameters(description = "Manage your admin entry. Seniors can also manage the entries of other admins.", usage = "/<command> [-o <admin name>] <clearips | clearip <ip> | setscformat <format> | clearscformat> | syncroles>")
+@CommandParameters(description = "Manage your Admin entry. Seniors can also manage the entries of other Admins.", usage = "/<command> [-o <admin name>] <clearips | clearip <IP> | setscformat <format> | clearscformat> | syncroles>")
```

Comment:
Should Admin be admin?

Reply:
Yeah I realized a bit after doing this that I should probably change it so that way whenever its referencing the rank it is capitalized and whenever its referencing admins in general its not